developers_chamber/qa/checks.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+from .base import QACheck, QAError
+
+
+class MissingMigrationsQACheck(QACheck):
+    """
+    Checks that make migrations command does not generate new migrations.
+    """
+    name = 'Check missing migrations'
+
+    def _run_check(self):
+        output = self._run_command('pydev make-migrations --dry-run')
+        last_line = output.split('\n')[-1]
+        if last_line != 'No changes detected':
+            raise QAError('Found missing migration(s)!', re.findall(r'\x1b.*?Migrations.+$', output, re.DOTALL)[0])
```

Comment:
That crazy regexp is to strip off output from wait-for-DB shell script, which pollutes the command output.

Reply:
I hope that django will not change the command oputput.

Do you now that django have check command for this https://docs.djangoproject.com/en/2.2/ref/django-admin/#check?

Reply:
Maybe for the future we should call only check, where can be implemented more our own check related to project and return its output

Reply:
For me, it makes sense to have generic checks here, and project specific checks implemented using the check framework.